sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import boto3\n",
+    "import time\n",
+    "from datetime import datetime\n",
+    "from sagemaker import get_execution_role, session"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set configurations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Region: ap-south-1\n",
+      "Role: arn:aws:iam::003394947794:role/service-role/SMClarifySageMaker-ExecutionRole\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Initialize sagemaker session\n",
+    "sagemaker_session = session.Session(boto3.Session())\n",
+    "\n",
+    "region = sagemaker_session.boto_region_name\n",
+    "print(f\"Region: {region}\")\n",
+    "\n",
+    "role = get_execution_role()\n",
+    "print(f\"Role: {role}\")\n",
+    "\n",
+    "bucket = sagemaker_session.default_bucket()\n",
+    "\n",
+    "prefix = \"sagemaker/DEMO-sagemaker-clarify-boto3\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download data\n",
+    "Data Source: [https://archive.ics.uci.edu/ml/machine-learning-databases/adult/](https://archive.ics.uci.edu/ml/machine-learning-databases/adult/)\n",
+    "\n",
+    "Let's __download__ the data and save it in the local folder with the name adult.data and adult.test from UCI repository$^{[2]}$.\n",
+    "\n",
+    "$^{[2]}$Dua Dheeru, and Efi Karra Taniskidou. \"[UCI Machine Learning Repository](http://archive.ics.uci.edu/ml)\". Irvine, CA: University of California, School of Information and Computer Science (2017)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "adult.data saved!\n",
+      "adult.test saved!\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sagemaker.s3 import S3Downloader\n",
+    "\n",
+    "adult_columns = [\n",
+    "    \"Age\",\n",
+    "    \"Workclass\",\n",
+    "    \"fnlwgt\",\n",
+    "    \"Education\",\n",
+    "    \"Education-Num\",\n",
+    "    \"Marital Status\",\n",
+    "    \"Occupation\",\n",
+    "    \"Relationship\",\n",
+    "    \"Ethnic group\",\n",
+    "    \"Sex\",\n",
+    "    \"Capital Gain\",\n",
+    "    \"Capital Loss\",\n",
+    "    \"Hours per week\",\n",
+    "    \"Country\",\n",
+    "    \"Target\",\n",
+    "]\n",
+    "if not os.path.isfile(\"adult.data\"):\n",
+    "    S3Downloader.download(\"s3://{}/{}\".format(\"sagemaker-sample-files\", \"datasets/tabular/uci_adult/adult.data\"), \"./\")\n",
```

Comment:
General: similarly the download method also has the `sagemaker_session` parameter.